modules/ROOT/pages/veevavault/veevavault-connector-studio.adoc

```diff
@@ -0,0 +1,177 @@
++
+. In the *Choose Global Type* window, expand *Connector Configuration*, click *Veeva Vault Config*, and click *OK*.
++
+. In the Veeva Vault Config window, in the *General* tab, enter the configuration values:
++
+image::veevavault/veevavault-connector-basic-authentication-general.png[]
++
+[%header%autowidth.spread]
+|===
+|*Field* |*Description*
+|*Name* |Enter a name for this configuration to reference it later.
+|*Vault URL* |Enter the Veeva Vault instance URL.
+|*Username* |Enter the Veeva Vault instance username.
+|*Password* |Enter the password that corresponds to the username.
+|*Version* |Enter the Veeva Vault instance version.
+|*Client Id* |Enter the client ID to send with each request in the header for tracking purposes, in the format: `\{organisation}-{company\|team}`. +
+The client ID is prefixed with "mulesoft" and has a suffix with "client-vaultconnector" internally, for example: `mulesoft-abcpharma-clinicalprod-client-vaultconnector`. 
+|===
++
+In the previous image, the placeholder values refer to a configuration file placed in the folder (`src/main/resources`) of your project. You can either hardcode your credentials into the global configuration properties or reference a configuration file that contains these values. +
+For simpler maintenance and better reusability of your project, using a configuration file is recommended. Keeping these values in a separate file is useful if you need to deploy to different environments, such as production, development, and QA, where your access credentials differ. +
+The following is an example snippet of the `configuration.yaml` file under the `src/main/resources/config` folder:
++
+[source,yaml,linenums]
+----
+	#Veeva Vault details
+	veeva:
+		vaultURL: "https://xxxx-xxxx.veevavault.com"
+		username: "someone@example.com"
+		password: "xxxxxxx"
+		version : "v19.1"
+		clientId: "mulesoft-abcpharma-clinicalprod-client-vaultconnector"
+----
++
+The connection configuration performs basic authentication, which accepts the required Vault URL, username, and password parameters to connect to the Veeva Vault instance to retrieve the session. The retrieved session is used for each operation configured in the Mule application for sending requests.
+. In the Veeva Vault Config window, *Advanced* tab, configure the burst API limit.
++
+image::veevavault/veevavault-connector-basic-authentication-advanced.png[]
++
+*Burst API Limit Profile*
++
+The burst API limit profile configuration to use if the specified burst API limit of the Veeva Vault API reaches the threshold of the configured value or is exceeded.
++
+If *Burst API Limit Exceed Profile* is enabled (default), then the connector evaluates the Veeva Vault API's response header (`X-VaultAPI-BurstLimitRemaining`) with the specified API Limit threshold value. If the remaining burst API limit reaches or exceeds this threshold value, the connector waits for the specified wait time to roll over the Burst Limit Remaining in five-minute windows.
```

Comment:
If something is a default, consider avoiding "if"...because...well, it's the default:
```suggestion
The default behavior for  *Burst API Limit Exceed Profile* is enabled, which means that the connector evaluates the Veeva Vault API's response header (`X-VaultAPI-BurstLimitRemaining`) with the specified API Limit threshold. If the remaining burst API limit reaches or exceeds this, the connector waits a specified time before rolling over the Burst Limit Remaining in five-minute windows.
```
To what does "in five-minute windows" refer: the rollover or the Burst Limit Remaining value?